STEERING-COMMITTEE.md

```diff
@@ -236,20 +244,20 @@ found in the [elections folder](./elections).
 
 ### Limitations on Company Representation
 
-No more than two seats may be held by employees of the same organization (or
-conglomerate, in the case of companies owning each other). Additionally, each
+No more than two contributing seats may be held by employees of the same organization
```

Comment:
Do we need to limit the selected seats too? Otherwise one company could in theory hold 4/7 seats (two contributing seats and the two selected seats)?

Reply:
Good question -- this would be done with the consent of the 2 other contributing seats and the end-user seat.  I'm willing to add this restriction, but I'm also thinking of times that we've approved an exception to these rules for the TOC, so I'm wondering if we want to hem ourselves in.

Reply:
Maybe just an overall limit of 3 seats per organization?  I'd love for it to be lower, but also dealing with the current reality.

Reply:
I don't think we need these limits anymore and should drop them and reintroduce them later when we have more vendors actively participating in the project.

Reply:
@dprotaso The problem with removing the limits is that if a malicious vendor ended up with a majority of seats (which could happen if not enough other people run), then there would be no way to reintroduce the limits because the vendor could always vote against reintroducing them, since this is the decision-making body that would make that change. 

The challenge with governance is that we need to anticipate the worst case scenarios and prevent them from happening.

Reply:
@geekygirldawn yeah good point

Reply:
Also, a good reason to have selected seats is to increase diversity.

Reply:
I think the question is, do we need to have company diversity limits on the selected seats?  I'm okay with adding that, sending a PR.

Reply:
Err, updating this PR.

Reply:
I like Josh's suggestion of an overall limit

Reply:
> I like Josh's suggestion of an overall limit
Same here

Reply:
I added an overall limit on seats.